src/containers/AdminCampaignList.jsx

```diff
@@ -42,12 +49,17 @@ class AdminCampaignList extends React.Component {
     createFromTemplateOpen: false,
     isCreating: false,
     campaignsFilter: {
-      isArchived: false
+      isArchived: false,
+      campaignTitle: ""
     },
     releasingInProgress: false,
     releasingAllReplies: false,
     releaseAllRepliesError: undefined,
-    releaseAllRepliesResult: undefined
+    releaseAllRepliesResult: undefined,
+    campaignDetailsForExport: [],
+    shouldShowExportModal: false,
+    shouldShowExportSnackbar: false,
```

Comment:
suggestion: use the standard `showExportModal`, `showExportSnackbar` variables, instead of `shouldShow`

Reply:
Also prefer `Dialog` over `Modal`, since we call everything dialogs in Spoke